volume/drivers/coprhd/coprhd.go

```diff
@@ -0,0 +1,262 @@
+package coprhd
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
```

Comment:
I think the comment is to use logrus as-is without renaming the import. (as part of cleanup we did that in the repo)